rl4co/models/zoo/matnet/policy.py

```diff
@@ -26,36 +41,171 @@ class MatNetPolicy(AutoregressivePolicy):
 
     def __init__(
         self,
-        env_name: str,
+        env: RL4COEnvBase,
```

Comment:
[Minor] usally we allow to pass both `str` and `RL4COEnvBase` as seen [here](https://github.com/ai4co/rl4co/blob/66622a2e7d616e72696904e1552cf7d61dad817f/rl4co/models/zoo/common/autoregressive/decoder.py#L75C1-L76C1)